.github/workflows/.build-job.yaml

```diff
@@ -20,11 +20,11 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
-      - name: Setup Java v17.x
+      - name: Setup Java v21.x
```

Comment:
Kanskje berre døpe om til Setup Java?

Reply:
Vil du ha det konsistent på alle workflows på tvers alle 4 repo? ;)

Reply:
På ingen måte viktig for meg, men sia du spør: ja 🤓

Reply:
Da er det nå det samme på tvers alle workflows i alle 4 repositories:

```yaml
      - name: Setup Java
        uses: actions/setup-java@v4
        with:
          distribution: temurin
          java-version: 21
          cache: gradle
```